### DISCUSSION TITLE
Does nuxt-font-loader prevent FOUT ?

### DISCUSSION THREAD

**Discussion by belfortf** [ivodolenc/nuxt-font-loader#22] [General]
I'm looking for a font strategy to deploy with Nuxt 3, currently having a lot of flash of unintended text. Thanks.

**Comment by ivodolenc** [comment]
> ... currently having a lot of flash of unintended text. 

Hi, please share more information because I can't tell you anything based on that.

This module automatically optimize fonts, so if you have any problems feel free to report a bug or provide some code example.
